src/util.py

```diff
@@ -123,6 +124,33 @@ def get_latest_ac_version_for_major_version(ac_repo, ac_major_version):
 MAVEN = "https://maven.mozilla.org/maven2"
 
 
+def get_latest_glean_version(gv_version, channel):
+    name = "geckoview"
+    if channel != "release":
+        name += "-" + channel
+    # A-C now builds against geckoview-omni
+    name += "-omni"
```

Comment:
Can we also reference that in the bug, that this change also changes the GV dependency to the Omni one? Just for clarity.